napari/benchmarks/benchmark_image_layer.py

```diff
@@ -47,7 +47,7 @@ def time_refresh(self, n):
         """Time to refresh view."""
         self.layer.refresh()
 
-    def _mem_layer(self, n):
+    def mem_layer(self, n):
         """Memory used by layer."""
         # Disabled because of __sizeof__ bug on the main branch and outdated asizeof in pympler
```

Comment:
All these comments are left over from the disabling!

Reply:
removed